### PR TITLE
One unsuccessful column instead of three

### LINKED ISSUE #1003: Show one `unsuccessful` column in the summary table, with a tooltip explaining the 3 possible reasons a test might be unsuccessful
When hovering over the `unsuccesful` heading of the summary table on https://bowtie.report, the modal that appears seems to have the content overflowing to the left.

https://github.com/bowtie-json-schema/bowtie/assets/75855708/f1afc446-4b78-4f1e-85c2-2d3c9105ec38

### PR DESCRIPTION
**Pull request by adwait-godbole**
Fixes #1003 

@Julian let me know of any improvements on this. Thanks!

----
📚 Documentation preview 📚: https://bowtie-json-schema--1226.org.readthedocs.build/en/1226/

### CODE REVIEW

File: frontend/src/components/Summary/ImplementationRow.tsx [hunk 1]
```diff
@@ -65,20 +65,20 @@ const ImplementationRow = ({
               <Container className="p-0">
                 <Row className="d-flex flex-column gap-2">
                   <Col>
-                    <span>
-                      <b style={{ fontSize: "1rem" }}>failed</b>:&nbsp;
+                    <b style={{ fontSize: "1rem" }}>failed:</b>&nbsp;
```
Comment (Julian) [inline frontend/src/components/Summary/ImplementationRow.tsx:68]: The manual `b` styling here was always a bit off to me, I think while we're here maybe can we change this to be labels? Maybe with a table. Or else maybe a `<hl>`? Or some other idea? I guess `hl` is most obvious to me.

File: frontend/src/components/Summary/ImplementationRow.tsx [hunk 2]
```diff
@@ -51,26 +57,45 @@ const ImplementationRow = ({
...
+      <OverlayTrigger
+        placement="left-end"
+        overlay={
+          <Popover style={{ border: "1px solid var(--bs-primary)" }}>
+            <Popover.Body>
+              <Container className="p-0">
+                <Row className="d-flex flex-column gap-2">
+                  <Col>
+                    <b style={{ fontSize: "1rem" }}>failed:</b>&nbsp;
+                    <span style={{ fontWeight: "500" }}>
```
Comment (Julian) [inline frontend/src/components/Summary/ImplementationRow.tsx:69]: Any reason you're not using the Bootstrap classes for font sizing and weights? https://getbootstrap.com/docs/5.3/utilities/text/#font-weight-and-italics

Comment (adwait-godbole) [reply frontend/src/components/Summary/ImplementationRow.tsx:69]: I wanted something that was just slightly less bolder than `fw-bold` and nothing in those options on the docs were satisfying the requirements so I made it 500.

File: frontend/src/components/Summary/ImplementationRow.tsx (hunk 1 above)
Comment (adwait-godbole) [reply frontend/src/components/Summary/ImplementationRow.tsx:68]: `<h1>` is too big for this purpose, going with `<h6>` instead.

**Comment by adwait-godbole** [comment]
@Julian I've pushed some changes and addressed the above comments. Can you take a look at the UI locally once and let me know if any other things that need to be changed. Thanks!

File: frontend/src/components/Summary/ImplementationRow.tsx (hunk 2 above)
Comment (adwait-godbole) [reply frontend/src/components/Summary/ImplementationRow.tsx]: It would've been easier if there was a `fw-semi-bold` instead. So had to do 500.

**Comment by adwait-godbole** [comment]
@Julian I've made some changes and addressed the above comments. Can run please run the UI locally once and confirm if any other things need to be changed ? Thanks!

**Comment by adwait-godbole** [comment]
I've made some changes and addressed the above comments. Can run please run the UI locally once and confirm if any other things need to be changed ? Thanks!

**Comment by adwait-godbole** [comment]
@Julian I am signing off for today but let me know if there's any other problem here.

**Comment by Julian** [comment]
This looks like an improvement I think, thanks!

**Pull request merged**